launchable/commands/record/build.py

```diff
@@ -159,8 +165,14 @@ def calc_branch_name(self):
                 refs = [ref for ref in show_ref.split("\n") if self.commit_hash in ref]
 
                 if len(refs) > 0:
-                    # e.g) ed6de84bde58d51deebe90e01ddfa5fa78899b1c refs/heads/branch-name
-                    self.branch = refs[0].split("/")[-1]
+                    # We assume the following values:
+                    # * ed6de84bde58d51deebe90e01ddfa5fa78899b1c refs/heads/branch/branch-name
+                    # * ed6de84bde58d51deebe90e01ddfa5fa78899b1c refs/remotes/origin/branch-name
+                    match = re.search('[a-f0-9]{40} refs/(heads|remotes/origin)/(.*)', refs[0])
+                    if match:
+                        self.branch = match.group(2)
```

Comment:
Note: https://docs.python.org/3/library/re.html#re.Match.group
if refs[0] was `refs/heads/branch/branch-name`

match.group(0) is `refs/heads/branch/branch-name`
match.group(1) is `heads`
match.group(2) is `branch/branch-name`